recipes-core/acrn/acrn-hypervisor.bb

```diff
@@ -16,12 +17,9 @@ PACKAGE_ARCH = "${MACHINE_ARCH}"
 DEPENDS += "python3-kconfiglib-native"
 DEPENDS += "${@'gnu-efi' if d.getVar('ACRN_FIRMWARE') == 'uefi' else ''}"
 
-do_configure() {
-	oe_runmake -C hypervisor oldconfig
-}
 
 do_compile() {
-	oe_runmake -C hypervisor
+	oe_runmake hypervisor
```

Comment:
The advantage of doing oldconfig and then explicitly building the hyperconfig directory is that you can alter the generated config instead of having to patch the default Kconfig.  Previously the top-level targets didn't work properly.